plugins/standard_tests/pyproject.toml

```diff
@@ -10,6 +10,7 @@ packages = [{include = "newhelm"}]
 python = "^3.10"
 # TODO: This probably shouldn't be here.
 perspective_api = {version = "0.1.1", optional = true}
+newhelm-openai = {version = "0.1.5", optional = true}
```

Comment:
Lets update this to version="*" and not be optional, as discussed in discord about the issue for perspective_api.